pkg/mounter/safe_mounter_host_process_windows.go

```diff
@@ -142,7 +142,6 @@ func (mounter *winMounter) Unmount(target string) error {
 		klog.V(2).Infof("remote server path: %s, local path: %s", remoteServer, target)
 		if hasDupSMBMount, err := smb.CheckForDuplicateSMBMounts(driverGlobalMountPath, target, remoteServer, mounter.volStatsCache); err == nil {
 			if !hasDupSMBMount {
-				remoteServer = strings.Replace(remoteServer, "UNC\\", "\\\\", 1)
```

Comment:
why remove this line? could you also paste the different behavior using powershell command and `os.Readlink` on a smb mount?

Reply:
powershell command is `c:\var\lib\kubelet\plugins\kubernetes.io\csi\file.csi.azure.com\cf2798d1325a0734ebe0a4c97cbee6ad5b8c1874e9b9c6d3d373d4e8dd016b81\globalmount -> UNC\f005e0c98fa2041c09d6df3.file.core.windows.net\pvc-572ab4d0-bc95-4df3-8de6-749726d87aa3`
and `os.Readlink` is `c:\var\lib\kubelet\plugins\kubernetes.io\csi\file.csi.azure.com\cf2798d1325a0734ebe0a4c97cbee6ad5b8c1874e9b9c6d3d373d4e8dd016b81\globalmount -> \\f005e0c98fa2041c09d6df3.file.core.windows.net\pvc-572ab4d0-bc95-4df3-8de6-749726d87aa3`

Reply:
pls squash commits, thx